### PR TITLE
update talks

### PR DESCRIPTION
**Pull request by wuji1mol**
加入主题：在线教育微信社群管理的实践和构想 by 轻课

### CODE REVIEW

**Comment by windmemory** [comment]
Welcome to Bot Friday club!

Please sign the CLA before we can process the PR further.

File: docs/_pages/talks.md [hunk 1]
```diff
@@ -58,3 +58,4 @@ Talks from our BOT5 members.
 | [S?E?](https://bot5.club/events/seminar-minutes-?-?/) | [郑譞](https://bot5.club/people/github_username) | [聊天机器人和自然语言的标注](https://www.bot5.club/talks/TBD) | Early November 2019 |
 | [S?E?](https://bot5.club/events/seminar-minutes-?-?/) | [李晶莹](https://bot5.club/people/github_username) | [聊天机器人聊论语](https://www.bot5.club/talks/TBD) | TBD |
 | [S?E?](https://bot5.club/events/seminar-minutes-?-?/) | [王冠](https://bot5.club/people/crownpku) | [Chatstack：搭建中文NLU的全流程用户交互系统](https://www.bot5.club/talks/TBD) | Late November 2019 |
+| [S?E?](https://bot5.club/events/seminar-minutes-?-?/) | [吴际](https://bot5.club/people/github_username) | [在线教育微信社群管理的实践和构想 by 轻课](https://www.bot5.club/talks/TBD) | Mid November 2019 |
```
Comment (windmemory) [inline docs/_pages/talks.md]: Please update the `github_username` to the real github name that you have.

Comment (windmemory) [reply docs/_pages/talks.md]: Your github name should be `wuji1mol`.

Comment (lijiarui) [inline docs/_pages/talks.md]: Thanks for the PR. We need the following change here:

1. This talk is S1E15, so you can change to S1E15 and move it to the first form.
2. https://bot5.club/people/github_username should change to your github link

**Comment by lijiarui** [comment]
duplicate with #95

**Pull request closed without merging**